src/stories/Library/card/card.scss

```diff
@@ -106,3 +63,102 @@
 .card__title {
   @include typography($typo__h3);
 }
+
+%card-variant--default {
+  width: 327px;
+
+  @include media-query__name($breakpoint__large-card) {
+    width: 551px;
+  }
+
+  .card__media {
+    // Odd aspect ratio, taken from the design.
+    aspect-ratio: 551/312;
+  }
+}
+
+.card {
+  @extend %card-variant--default;
+
+  position: relative;
+  max-width: 100%;
+  word-break: break-word;
+
+  img {
+    height: 100%;
+    width: 100%;
+    object-fit: cover;
+    object-position: center;
+  }
+
+  a {
+    text-decoration: none;
+  }
+}
+
+%card-variant--large,
+.card[data-variant="large"] {
+  @extend %card-variant--default;
+}
+
+%card-variant--x-large,
+.card[data-variant="x-large"] {
+  width: 200px;
+
+  @include media-query__name($breakpoint__large-card) {
+    width: 440px;
+  }
+
+  &.card--has-no-image {
+    // We only want the page fold on 1:1 view-modes.
+    @extend %identity-pagefold;
+  }
+
+  .card__placeholder-text {
+    @include typography($typo__card-placeholder);
+  }
+
+  .card__media {
+    aspect-ratio: 1/1;
+  }
+}
+
+%card-variant--small,
+.card[data-variant="small"] {
+  @extend %card-variant--x-large;
+
+  width: 206px;
+
+  .card__tags,
+  .card__media {
+    margin-bottom: $s-xs;
+  }
+
+  // On desktop sizes this small, there is no space for placeholder text, and
+  // extra tags, but we can still show it for screenreaders.
+  @include media-query__name($breakpoint__large-card) {
+    width: 206px;
+
+    .card__placeholder-text {
+      font-size: 0;
+    }
+
+    .card__tag {
+      @include typography($typo__byline-tags--small);
+    }
+
+    .card__title {
+      @include typography($typo__h4);
+    }
+  }
+}
```

Comment:
I dont fully understand the implications of the combined css, or if i'm hitting the all the correct parts. Possibly it works just as intended. 

So i'll just highlight that a small card with no image, has no text inside it the identityplaceholder when above the $breakpoint__large-card, which i understand to be the medium breakpoint of 1024px. 

If this is intended then i guess all is well, but it seemed strange to me that the text for would disappear on larger screens for seemingly no reason.

Reply:
This is correct and intended
On mobile, the "small" card is the same size as the x-large, so there is actually space for the placeholder text, as it also becomes smaller.
On desktop, the placeholder text will become bigger, but the small box doesnt, so there's no space.

This is also something I want to take with DDF during demo.